.github/workflows/ci-dependencies-mac.yml

```diff
@@ -0,0 +1,40 @@
+name: SMARTS CI Dependencies Mac
+
+on:
+  schedule:
+    - cron: '0 23 * * 1'
+      # Runs at 11.00pm, UTC, every Monday
```

Comment:
I think this could get into a race condition with the `SMARTS CI Base Tests Mac` action. This action will either need to trigger the `SMARTS CI Base Tests Mac` action or run at some time comfortably before.

Reply:
Modified workflows such that completion of Mac requirements update will trigger Mac base tests.